README.md

```diff
@@ -54,7 +54,29 @@ Each of these tools have configuration specific to their usage in a monorepo, wh
 
 If you plan to use any of the following, please see the style guide for how to set them up:
 
-- [next.js](https://github.com/Thinkmill/monorepo#nextjs)
+### Installing new packages
+
+When you install new packages, you will need to determine if they are intended to be used by the monorepo as a whole, or by an individual package.
+
+If it is intended for use by the monorepo as a whole, add the package to the root `package.json` and run `yarn` to install.
+
+If it is intended for use by an individual package, add it to that's `package.json` and run `yarn` to install.
```

Comment:
Should we also show the command for doing these?

```
If it is intended for use by the monorepo as a whole, add the package with `yarn add -W package-name`.

If it is intended for use by an individual app, add it there with `yarn add --cwd=apps/foo package-name`.
```

Reply:
We actually vaguely recommend manually adding packages to `package.json` and then running `yarn` to do an install, as yarn occasionally does things differently for this and `add`, with the not-add being more correct for how we want things to be installed/locked.